velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMethods.java

```
@@ -0,0 +1,371 @@
+        sendMessage(player, str);
+      }
+      return true;
+    }
+    return false;
+  }
+
+  @Override
+  public boolean callCMD(Object player, String cmd) {
+    Punishment pnt;
+    if (Universal.get().isMuteCommand(cmd.substring(1))
+        && (pnt = PunishmentManager.get().getMute(UUIDManager.get().getUUID(getName(player)))) != null) {
+      for (String str : pnt.getLayout()) {
+        sendMessage(player, str);
+      }
+      return true;
+    }
+    return false;
+  }
+
+  @Override
+  public String parseJSON(InputStreamReader json, String key) {
+    JsonElement element = new JsonParser().parse(json);
+    if (element instanceof JsonNull) {
+      return null;
+    }
+    JsonElement obj = ((JsonObject) element).get(key);
+    return obj != null ? obj.toString().replaceAll("\"", "") : null;
+  }
+
+  @Override
+  public String parseJSON(String json, String key) {
+    JsonElement element = new JsonParser().parse(json);
+    if (element instanceof JsonNull) {
+      return null;
+    }
+    JsonElement obj = ((JsonObject) element).get(key);
+    return obj != null ? obj.toString().replaceAll("\"", "") : null;
+  }
+
+  @Override
+  public Boolean getBoolean(Object file, String path) {
+    return ((ConfigurationNode)file).getNode(path).getBoolean();
+  }
+
+  @Override
+  public String getString(Object file, String path) {
+    return ((ConfigurationNode)file).getNode(path).getString();
+  }
+
+  @Override
+  public Long getLong(Object file, String path) {
+    return ((ConfigurationNode)file).getNode(path).getLong();
+  }
+
+  @Override
+  public Integer getInteger(Object file, String path) {
+    return ((ConfigurationNode)file).getNode(path).getInt();
+  }
+
+  @Override
+  public List<String> getStringList(Object file, String path) {
+    try {
+      return ((ConfigurationNode)file).getNode(path).getList(TypeToken.of(String.class));
+    } catch (ObjectMappingException e) {
+      e.printStackTrace();
+      return null;
```

Rethrow as an unchecked exception and fail-fast rather than returning null and causing further NPEs down the line.

---

Unless I'm mistaken - I cannot rethrow an ObjectMappingException as an UncheckedIOException? Unless of course you mean a different unchecked exception

---

RuntimeException or any subclass which makes sense is fine:
```java
try {
  // configurate which throws checked exceptions
} catch (ObjectMappingException ex) {
  throw new RuntimeException(ex);
}
```